src/app/plugins/addroles.plugin.ts

```diff
@@ -53,3 +64,8 @@ export class AddRolesPlugin extends Plugin {
     }
   }
 }
+
+interface IRoleEmoji{
+  emojiName: string,
+  emoji: undefined | GuildEmoji
```

Comment:
use `Maybe<GuildEmoji>` instead

Reply:
nevermind, instead just make it `GuildEmoji` then cast GuildEmoji when you [retrieve it](https://github.com/VijayStroup/lion/blob/13fc2364601d1bbd470c34039f8be30ed6a25cd3/src/app/plugins/addroles.plugin.ts#L29)